### DISCUSSION TITLE
how to access any JavaScript object in browser

### DISCUSSION THREAD

**Discussion by kangaroolab** [borkdude/scittle#84] [General]
hi, besides access window and document in browser, can you please advise how to access other JavaScript object?

e.g. if I have a object called myeditor, which I can access in browser inspector, how can I access it in scittle?

(. js/myeditor -attr)

this doesn't work.

**Comment by borkdude** [comment]
Normally I'd write `(.-attr js/myeditor)` 

Does `globalThis.myeditor.attr` work in the browser console?

**Comment by kangaroolab** [comment]
got it...thanks
